pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -31,12 +36,12 @@ func getDSResourceName() string {
 
 func TestListDaemonSets(t *testing.T) {
 	ns := api.NamespaceAll
-	c := &testClient{
-		Request: testRequest{
+	c := &simple_testclient.TestClient{
```

Comment:
I will smilarly rename simple_testclient#TestRequest to simple_testclient#Request